xml/System.Text.Json/JsonDocument.xml

```diff
@@ -65,11 +65,13 @@
         <Parameter Name="readerOptions" Type="System.Text.Json.JsonReaderOptions" />
       </Parameters>
       <Docs>
-        <param name="utf8Json">To be added.</param>
-        <param name="readerOptions">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="utf8Json">JSON text to parse.</param>
```

Comment:
cc @bartonjs for review since these haven't made it to https://docs.microsoft.com/en-us/dotnet/api/system.text.json.jsondocument?view=netcore-3.0

Reply:
I updated the PR. Added missing remarks and some other exceptions.